server/services/RunService.js

```diff
@@ -376,7 +382,37 @@ async function configureRuns(
                 };
             });
 
-            await db.Run.bulkCreate(dbRuns);
+            const newRuns = await db.Run.bulkCreate(dbRuns);
+
+            // Add tester to the runs that match on ApgExample, ATVersion, and BrowserVersion
```

Comment:
We only want to copy over the testers if the test_version changes, otherwise we run through all this code unnecessarially because you will never find a matchIndex (because if you aren't changed the test_version, there is a uniqueness to the "browser_version_to_at_version.id" and "apg_example.name").

So maybe should should block off all this code based on whether there is a test_version change?

Reply:
Good idea! I'll implement this.